CODEOWNERS

```diff
@@ -258,6 +258,7 @@ homeassistant/components/weather/* @fabaff
 homeassistant/components/weblink/* @home-assistant/core
 homeassistant/components/websocket_api/* @home-assistant/core
 homeassistant/components/wemo/* @sqldiablo
+homeassistant/components/withings/* @vangorra
```

Comment:
This excludes all tests from the code coverage.
Please only exclude files that don't have tests.

Reply:
This is the CODEOWNERS file 👍

Reply:
🤦‍♂ I had an awesome week. Sorry about that.